cardano-node/src/Cardano/Node/Tracing/Tracers/ChainDB.hs

```diff
@@ -1522,6 +1522,8 @@ instance ( StandardHash blk
              " This is most likely an expected change in the serialization format,"
```

Comment:
(can't comment there)
Probably this code above would benefit from some spaces in between:
```
"Invalid snapshot " <> showT snap <> showT failure <> context
```

Reply:
Agreed.